content/blogposts/2022/release-4.2.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Sourcegraph 4.2 release"
+publishDate: 2022-11-29T10:00-07:00
+description: "Sourcegraph 4.2 introduces the new audit log, executor secrets for server-side Batch Changes, and Smart Search to make it easier to find the search results you're looking for."
+tags: [blog, release]
+slug: "release/4.2"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.2/sourcegraph-4-2-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.2/sourcegraph-4-2-hero.png
+changelogItems:
+  - description: The search input has a new recent searches button, and recent searches can be cycled via the up/down arrow keys for quick access to previous searches.
+    url: https://github.com/sourcegraph/sourcegraph/pull/44544
+    category: Search
+  - description: "When the `content:` filter is used in a query, it now behaves more predictably by only searching file contents. Previously, file contents, paths, and repos were searched. However, as before, if `type:` is also set, the `content:` filter will search for results of the specified `type:`."
+    url: https://github.com/sourcegraph/sourcegraph/pull/43442
+    category: Search
+  - description: When rendering a file which is backed by Git LFS, Sourcegraph now displays a page that links directly to the file on the codehost. Previously we rendered the LFS pointer.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43686
+    category: Search
+  - description: Code Insights data points that do not contain any results will display zero instead of being omitted from the visualization for clarity. This only applies to insight data created after 4.2.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43166
+    category: Code Insights
+  - description: "A new experimental GraphQL query, `permissionsSyncJobs`, now lists the states of recently completed permissions sync jobs and the state of each provider to check that syncing is working as intended. The TTL of entries can be configured with `authz.syncJobsRecordsTTL`."
+    url: https://github.com/sourcegraph/sourcegraph/pull/44387
+    category: API
+  - description: Repositories can now be ordered by size on the repo admin page to make it easy to find the largest synced repos.
```

Comment:
@iskyOS - should repo admin page be repository admin page? And largest synced repositories?